numpy_financial/tests/test_financial.py

```diff
@@ -26,7 +26,7 @@
 def assert_decimal_close(actual, expected, tol=Decimal("1e-7")):
     # Check if both actual and expected are iterable (like arrays)
     if hasattr(actual, "__iter__") and hasattr(expected, "__iter__"):
-        for a, e in zip(actual, expected):
+        for a, e in zip(actual, expected, strict=False):
```

Comment:
I haven't looked at this test in several years, but has something changed so that we required ``strict=False``?

Reply:
well, ruff changed, and added this `B905` rule https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/

Reply:
Ok, that shouldn't be a problem. As this is a test, and ``len(actual)`` should be the same as ``len(expected)``, could we set ``strict=True``?

Reply:
Setting `strict=True` indeed enforces that the length should be equal, otherwise it'll raise a `ValueError`. So it's basically a "free" `if len(actual) != len(expected): raise ValueError`.

Reply:
Oh woops nevermind I didn't notice that `strict=False`.